lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -295,6 +295,9 @@ def populate_api_routes(webapp, app):
     webapp.mapper.resource('group', 'groups', path_prefix='/api')
     webapp.mapper.resource_with_deleted('quota', 'quotas', path_prefix='/api')
 
+    webapp.mapper.connect('/api/cloud/authz/', action='index', controller='cloudauthz')
+    webapp.mapper.connect('/api/cloud/authz/create', action='create', controller='cloudauthz')
```

Comment:
So here you can specify actions, specify the one above to be a GET and this one to be a POST. There are examples in this file.

Reply:
Please check the following PR: https://github.com/galaxyproject/galaxy/pull/6876